README.md

```diff
@@ -33,8 +33,8 @@
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/219087606-bc6124b0-14ff-4f6a-b2e6-9cbf99e99575.png)

**[Optional]**

- [ ] Kindly remove the unnecessary sections from your application such as shown in the image above to make your readme file clean, professional and descriptive :blush: